media-autobuild_suite.bat

```diff
@@ -414,6 +414,28 @@ if %buildother265%==2 set "other265=n"
 if %buildother265% GTR 2 GOTO other265
 if %deleteINI%==1 echo.other265=^%buildother265%>>%ini%
 
+:svthevc
+if %svthevcINI%==0 (
+    echo -------------------------------------------------------------------------------
+    echo -------------------------------------------------------------------------------
+    echo.
+    echo. Build SVT-Hevc? [H.265 encoder]
+    echo. 1 = Yes
+    echo. 2 = No
+    echo.
+    echo. Requires at least an Intel first generation core or AMD Bulldozer.
```

Comment:
Add note if this is required to build or to execute.

Reply:
Okay.